sdk/react/utils.js

```diff
@@ -215,6 +215,9 @@ var Utils = {
       /* Unknown error - TODO add to language files */ 
       case 28:
         return "An unknown error occurred";
+      /* GeoBlocking access denied - TODO add to language files */
+      case 31:
```

Comment:
were are the cases 29 and 30?

Reply:
I think there is an mistake. We have 31 errors in class OoyalaException (sdk), but here i see only 28 errors. I will add missed error codes.